python/python/glide/async_commands/core.py

```diff
@@ -362,6 +362,11 @@ class InsertPosition(Enum):
     AFTER = "AFTER"
 
 
+class FlushMode(Enum):
```

Comment:
Add a doc to this enum.
Ensure that it is exported to the user

Reply:
how to make sure it's exported to the user?

Reply:
I think you need to update `__init__` or so
please, check merged python PRs on upstream